.circleci/config.yml

```diff
@@ -9,21 +9,21 @@ orbs:
 parameters:
   beta_build:
     type: boolean
-    default: false 
+    default: false
```

Comment:
Sorry for the trailing whitespace noise. I thought I'd leave it in here since this PR is quite small.